cl/singlenode/payloadstore/postgres.go

```diff
@@ -41,6 +42,16 @@ func NewPostgresRepository(ctx context.Context, dsn string, logger *slog.Logger)
 		return nil, fmt.Errorf("failed to ping postgres: %w", err)
 	}
 
+	go func() {
```

Comment:
I dont think this is required. We are constantly writing to postgres every 200ms at the very least. Not sure what this ping will do that a write will not.